packages/x-components/src/x-modules/facets-next/entities/__tests__/equatable-filter.spec.ts

```diff
@@ -1,20 +0,0 @@
-import { createNextSimpleFilter } from '../../../../__stubs__/filters-stubs.factory';
```

Comment:
Equatable filter has been removed. The initial idea of this was to have a way to compare the same kind of filters. The editable number range filter was supposed to have a static Id, so we had to compare them using the range values, but we change it in this PR, and its id is now modified depending on the values.  So now we can just compare the filters id.